ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -124,10 +124,14 @@ synchronized VungleBannerAdapter getBannerRequest(@NonNull String placementId, @
         VungleBannerAdapter bannerRequest = mVungleBanners.get(placementId);
         if (bannerRequest != null) {
             String activeUniqueRequestId = bannerRequest.getUniquePubRequestId();
-            if (activeUniqueRequestId != null && activeUniqueRequestId.equals(requestUniqueId) || activeUniqueRequestId == null && requestUniqueId == null) {
-                Log.d(TAG, "Seems like Refresh: activeUniqueId: " + activeUniqueRequestId + " ###  RequestId: " + requestUniqueId);
-            } else {
-                Log.d(TAG, "Does NOT seems like Refresh: activeUniqueId: " + activeUniqueRequestId + " ###  RequestId: " + requestUniqueId);
+            Log.d(TAG, "activeUniqueId: " + activeUniqueRequestId + " ###  RequestId: " + requestUniqueId);
+            if (activeUniqueRequestId == null) {
```

Comment:
is it ok to break any existing integration?
Could we check if mVungleBanners has already this placement id and fail will old integration code only in this case?

Reply:
https://github.com/googleads/googleads-mobile-android-mediation/pull/212#discussion_r395244342
Admob strongly recommend us to limit the refresh logic.

Reply:
Shouldn't we fail if the publisher doesn't pass extras on the step of banner creation then?

Reply:
My understanding is for the first time we should load ad if none extra passed or uniqueId set to null, but for the second request(refresh/new AdRequest) we should break it and reminder pubs to correctly integration Vungle banner in logs.
It's confusing that AdMob didn't comment iOS to work on this similar change.

Reply:
Got it, just worrying that somebody might not test the refresh, could we enable VungleBanner refresh for that case?

Reply:
I think its better that only depends on whether pubs enabled it on AdMob side.